### PR TITLE
fix: add the strings similar to google home

### LINKED ISSUE #1661: Improve the strings and make it consistent with google home
**Actual Behaviour**

Currently the strings are not consistent with the google home app

**Expected Behaviour**

Make the strings consistent with the google home app

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1661

### CODE REVIEW

**Comment by batbrain7** [comment]
@arundhati24 please check the string changes and tell if anything needs to be changed

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -366,12 +366,14 @@
...
+    <string name="device_help">Choose the Wi-Fi network you would like to use with your smart speaker.</string>
     <string name="connecting_device">Connecting to your device...</string>
     <string name="device_name">SUSI.AI</string>
     <string name="connection_success">Connected Successfully !</string>
-    <string name="choose_wifi">Choose a Wi-Fi</string>
+    <string name="choose_wifi">Connect to Wi-Fi</string>
     <string name="enter_password">Enter the Password for \n</string>
-    <string name="enter_password_mail">Enter the password for your account \n</string>
+    <string name="thanks_wifi">Thanks for entering the Wi-Fi credentials\n</string>
+    <string name="enter_password_mail">This is the final step, please enter your SUSI.AI account\'s password to successfully connect to the speaker</string>
```
Comment (arundhati24) [inline app/src/main/res/values/strings.xml]: account's password?

Comment (arundhati24) [reply app/src/main/res/values/strings.xml]: No full stop?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -366,12 +366,14 @@
...
     <string name="set_up">SET UP</string>
+    <string name="device_help">Choose the Wi-Fi network you would like to use with your smart speaker.</string>
     <string name="connecting_device">Connecting to your device...</string>
     <string name="device_name">SUSI.AI</string>
     <string name="connection_success">Connected Successfully !</string>
-    <string name="choose_wifi">Choose a Wi-Fi</string>
+    <string name="choose_wifi">Connect to Wi-Fi</string>
     <string name="enter_password">Enter the Password for \n</string>
-    <string name="enter_password_mail">Enter the password for your account \n</string>
+    <string name="thanks_wifi">Thanks for entering the Wi-Fi credentials\n</string>
```
Comment (arundhati24) [inline app/src/main/res/values/strings.xml]: No punctuation?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -366,12 +366,14 @@
     <string name="location_access">Provide Location Access</string>
     <string name="location_access_message">Please turn on the device\'s location as it is required to detect the nearby wifi networks available.</string>
     <string name="set_up">SET UP</string>
+    <string name="device_help">Choose the Wi-Fi network you would like to use with your smart speaker.</string>
     <string name="connecting_device">Connecting to your device...</string>
     <string name="device_name">SUSI.AI</string>
     <string name="connection_success">Connected Successfully !</string>
-    <string name="choose_wifi">Choose a Wi-Fi</string>
+    <string name="choose_wifi">Connect to Wi-Fi</string>
```
Comment (arundhati24) [inline app/src/main/res/values/strings.xml]: same

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (arundhati24) [reply app/src/main/res/values/strings.xml]: Could the string resource name be a bit more standard?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (arundhati24) [reply app/src/main/res/values/strings.xml]: Could the string resource name be a bit more meaningful?

**Comment by batbrain7** [comment]
@arundhati24 suggestions?

**Comment by batbrain7** [comment]
@arundhati full stops are not present in Google home if you checked it you would have known also , all the strings except one are consistent to Google Home, I will modify the one string that remains

**Comment by mariobehling** [comment]
Merging this as WIP.

**Pull request merged**